src/main/java/com/seiama/javaducks/controller/JavadocController.java

```diff
@@ -152,4 +155,35 @@ public ResponseEntity<?> serveFavicon(@PathVariable final String project) {
         .build();
     }
   }
+
+  @GetMapping("/{project:[a-z]+}/")
```

Comment:
you mixed up the mappings, this one should have no trailing slash and the one below should have one, right now /paper/ redirects to /paper//